lib/tasks/e2e_seed.rake

```diff
@@ -0,0 +1,14 @@
+# Thanks to https://stackoverflow.com/questions/19872271/adding-a-custom-seed-file/31815032#31815032
+SEED_DIR = Rails.root.join("spec/support/seeds/e2e/")
+
+namespace :db do
+  namespace :seed do
+    desc "Seed data for e2e tests"
+    task :e2e, [:file] => :environment do |_t, args|
+      return if Rails.env.production?
```

Comment:
Folks should be doing this in the test seed files, but I added an extra guard here for production in case it's omitted in the e2e test seed file.

Reply:
@nickytonline You wanted to extract the `Seeder` class anyway, we probably could move this in there. We could e.g. pop it into `initialize`:

```ruby
def initialize
  fail "Attempting to seed production environment, aborting!" if Rails.env.production?

  @counter = 0
end
```

Reply:
Never mind, you already extracted the class. Maybe you want to add it there?